test/test.js

```diff
@@ -273,10 +273,10 @@ test('Should not throw within different contexts', (t) => {
         host: '127.0.0.1',
         namespace: 'test1'
       })
-      .register(fastifyRedis, {
-        host: '127.0.0.1',
-        namespace: 'test2'
-      })
+    instance.register(fastifyRedis, {
```

Comment:
was this chain call fail?

Reply:
It was definitely failing before I patched avvio. There should be no need for this change.

Reply:
Reverted this change